src/framework/components/element/component.js

```diff
@@ -87,6 +87,7 @@ Object.assign(pc, function () {
      * @property {Number} materialAsset The id of the material asset to use when rendering an image. Only works for {@link pc.ELEMENTTYPE_IMAGE} types.
      * @property {pc.Material} material The material to use when rendering an image. Only works for {@link pc.ELEMENTTYPE_IMAGE} types.
      * @property {pc.Vec4} rect Specifies which region of the texture to use in order to render an image. Values range from 0 to 1 and indicate u, v, width, height. Only works for {@link pc.ELEMENTTYPE_IMAGE} types.
+     * @property {Function} rtlReorder Optional function to reorder the text for RTL languages.
```

Comment:
Are there going to be multiple ways of rtl reordering? If not I guess we could make this a boolean?

Reply:
The issue with making it a boolean is then we have to include the algorithm for RTL/bidi in the engine code itself, right?

Reply:
Yes we would. I don't know if that's a problem though I'm just curious. Is it like a huge library?